Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -7010,6 +7015,113 @@ local Item5 = cItem(E_ITEM_DIAMOND_CHESTPLATE, 1, 0, "thorns=1;unbreaking=3");
 				},
 			},
 		},
+		cBookContent =
+		{
+			Desc = [[
+This class contains the information for a signed or writeable book: The author, title and the pages. A page of the writeable book is a simple string. For a signed book it can be a json string. For the json string use {{cCompositeChat}} to create a page and then the function {{cCompositeChat#CreateJsonString_1|CreateJsonString}} to get the json string.
```

Comment:
Ideally accept and store json for both and then in the protocol handler convert whatever is not supported into a supported form. So that if MC decides to support json in the future, we don't have trouble upgrading stuff.